src/main/resources/MacIntegrationsBundle.properties

```diff
@@ -1 +1,2 @@
-org.cryptomator.macos.keychain.displayName=macOS Keychain
\ No newline at end of file
+org.cryptomator.macos.keychain.displayName=macOS Keychain
+org.cryptomator.macos.keychain.touchIdDisplayName=Touch ID
```

Comment:
I don't think, this should be localizable. Apple uses Touch ID in support documents for different languages (e.g. french https://support.apple.com/fr-fr/guide/mac-help/mchl16fbf90a/mac)

Reply:
That's not entirely true. In a few languages, it's being localized. I think it should be kept here, we're doing the same on iOS.